src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -23,7 +24,10 @@ private Settings(IConfigurationSource source)
                 throw new ArgumentNullException(nameof(source));
             }
 
-            Exporter = source.GetString(ConfigurationKeys.Exporter);
+            Exporter = source.GetString(ConfigurationKeys.Exporter) ?? "otlp";
+            OtlpExportProtocol = GetExporterOtlpProtocol(source);
+            OtlpExportEndpoint = GetExporterOtlpEndpoint(source, OtlpExportProtocol);
```

Comment:
There's some non-trivial logic in these 2 new methods. Do we think that it's worth introducing some unit tests, even though some of that code will go away when the SDK is using the correct port by default?

Reply:
I hope that most of the changes will be removed shortly. It was the only reason to not include the UnitTests.

Added, both scaffolding for the Managed.Tests and tests.